From 98e793d9f2250e7c1f9f1eb5dfd616a6a8829e9a Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 3 Sep 2018 11:12:27 -0700
Subject: [PATCH] gen_def

This is partial port because half of this patch is already present in our
patches

Upstream-Status: Backport [https://github.com/apache/serf/commit/8f07c9626106873a594fd07e7c257e9080d8660b]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 SConstruct | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SConstruct b/SConstruct
index 7bb4a85..01ec245 100644
--- a/SConstruct
+++ b/SConstruct
@@ -157,6 +157,8 @@ env = Environment(variables=opts,
                   CPPPATH=['.', ],
                   )
 
+gen_def_script = env.File('build/gen_def.py').rstr()
+
 env.Append(BUILDERS = {
     'GenDef' : 
       Builder(action = '"%s" "%s" $SOURCES > $TARGET' % (sys.executable, gen_def_script,),
